bin/yaml/libraries.yaml

```diff
@@ -377,9 +377,17 @@ libraries:
       - 2.3.8
       type: github
     eastl:
+      after_stage_script:
+      - mkdir build
+      - cd build
+      - /opt/compiler-explorer/cmake/bin/cmake ..
+      - "[ -e ./_deps ] && cp -r ./_deps/eabase-src/include/Common/EABase ../include/EABase"
```

Comment:
The `[ -e ./_deps ]` part is just so the `cp` doesn't fail on versions which do not use `FetchContent`. Again, if this feels wrong to anyone I'm very open to alternatives.

Reply:
I've removed this now that we only do this step on versions where the `cp` is guaranteed to work.